torchfix/visitors/internal/__init__.py

```diff
@@ -0,0 +1,33 @@
+import libcst as cst
+from ...common import TorchVisitor, LintViolation
+
+
+class TorchScopedLibraryVisitor(TorchVisitor):
+    """
+    Suggest `torch.library._scoped_library` for PyTorch tests.
+    """
+
+    ERROR_CODE = "TOR901"
+    MESSAGE = (
+        "Use `torch.library._scoped_library` instead of `torch.library.Library` "
+        "in PyTorch tests files. See https://github.com/pytorch/pytorch/pull/118318 "
+        "for details."
+    )
+
+    def visit_Call(self, node):
+        qualified_name = self.get_qualified_name_for_call(node)
+        if qualified_name == "torch.library.Library":
+            position_metadata = self.get_metadata(
+                cst.metadata.WhitespaceInclusivePositionProvider, node
+            )
+
+            self.violations.append(
```

Comment:
what limits this rule to files under test/ ?

Reply:
The rules is disabled by default. We can enable it for test/ in the CI config in pytorch/pytorch.